docs/Translation.md

```diff
@@ -12,27 +12,37 @@ The react-admin interface uses English as the default language. But it also supp
 To handle translations, the `<Admin>` component supports:
 
 - a `locale` prop expecting a string ('en', 'fr', etc), and
-- a `messages` prop, expecting a dictionary object.
+- a `messagesProvider` prop, expecting an `messagesProvider` as explained below.
```

Comment:
I liked the `i18nProvider` name. The term *messages* isn't specific enough IMO (it can relate to notifications in many people's mind).